src/paho/mqtt/__init__.py

```diff
@@ -1,4 +1,10 @@
-__version__ = "1.2.2"
+__version__ = "1.3.0.dev0"
```

Comment:
There are plenty of tools out there for doing versioning a bit more sensibly.  What I use for some of my projects is `versioneer` -- https://github.com/warner/python-versioneer.

Versions are parsed from git tags, and the version string includes the closest "tag" with the number of commits added since that tag, with the beginning of the commit hash that the install was cut from.  Pretty nice, I think.